events/break-free/index.html

```diff
@@ -72,6 +72,8 @@
             {{ description }}
           </div>
         </script>
+
+  <script type="text/javascript" src="js/action-kit-service.js"></script>

```

Comment:
`moment.js` and `icon-service.js` are used by `action-kit-service.js`, but only if a campaign string was passed in. So they won't cause any errors here, since the code never gets to the point where they are called. But we always need `action-kit-service.js` because it's called by `script.js` and we'll get a console error if we omit it.

I can see that this is pretty confusing, not knowing which files are needed or not for each `index.html`. One solution would be to always include everything, or a better solution (involving more work/refactoring) could be to use a JavaScript bundler like [Browserify](http://browserify.org/) to bundle all the JS files into one.